hello-jni/app/build.gradle

```diff
@@ -68,6 +76,6 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:25.2.0'
-    compile 'com.android.support.constraint:constraint-layout:1.0.1'
+    compile 'com.android.support:appcompat-v7:25.4.0'
```

Comment:
Could use global properties here?

See: https://developer.android.com/studio/build/gradle-tips.html#configure-project-wide-properties

(And yeah, I just noticed that I specify buildToolsVersion in the sample. I'll remove it :)

Reply:
this is good point, will use this global feature in multi-module project environment.  This one is one module project, keep it in one place for simplicity case.